### DISCUSSION TITLE
Column visual select + delete

### DISCUSSION THREAD

**Discussion by danjenson** [helix-editor/helix#6363] [Q&A]
### Summary

I can't seem to select columns. In (n)vim, you could do this with shift+V. This is useful for formatting files that don't have auto formatters. I can't seem to isolate a block of columns to delete.

### Reproduction Steps

I tried to use variants of X and &, but to no avail.

### Platform

Void Linux

### Terminal Emulator

Alacritty

### Helix Version

helix 22.12

**Comment by Carrotstrip** [comment]
You can create another cursor underneath the current cursor with shift+c as many times as you like, and then v to enter select mode, then normal cursor navigation will select a block of columns.
